pkg/sysprof/network_darwin.go

```diff
@@ -5,10 +5,27 @@ type SPNetworkDataType struct {
 }
 
 type NetworkDataType struct {
-	DNS       DNS    `json:"DNS"`
-	Interface string `json:"interface"`
+	DNS       DNS     `json:"DNS"`
+	Interface string  `json:"interface"`
+	Proxies   Proxies `json:"proxies"`
```

Comment:
nit: `json:”Proxies”` for consistency with other json fields

Reply:
Interesting! I didn't know that JSON marshalling will use a case-insensitive match if it can't find an exact match, which is why this still worked despite not having the right capitalization. Fixed.

Reply:
I've left `interface` lowercase because that is how it comes in from `system_profiler`:

```console
$ system_profiler SPNetworkDataType -json | jq -r '.SPNetworkDataType[0]|keys[]' | sort
DNS
Ethernet
IPv4
IPv6
Proxies
_name
dhcp
hardware
interface
ip_address
sleep_proxies
spnetwork_service_order
type
```